baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -59,12 +60,26 @@ Additional metadata MAY be appended to values in the form of property set, repre
 Leading and trailing OWS is allowed but MUST be trimmed when converting the header into a data structure.
 
 ## Limits
-1. Maximum number of name-value pairs: `180`.
-2. Maximum number of bytes per a single name-value pair: `4096`.
-3. Maximum total length of all name-value pairs: `8192`.
+1. Maximum number of list-members: `180`.
+2. Maximum number of bytes per list-member: `4096`.
```

Comment:
bytes -> characters? We never say ASCII anywhere

Reply:
I mean we say ASCII when we say what smbols are allowed, but if somebosy will read it into the Unicode, we may want to talk about characters

Reply:
I prefer bytes since it is a more well-defined term. We could adopt the language of "octets" similar to the cookie header if you would prefer?

From cookie header:

```
NOTE: Despite its name, the cookie-string is actually a sequence of
   octets, not a sequence of characters.  To convert the cookie-string
   (or components thereof) into a sequence of characters (e.g., for
   presentation to the user), the user agent might wish to try using the
   UTF-8 character encoding [RFC3629] to decode the octet sequence.
   This decoding might fail, however, because not every sequence of
   octets is valid UTF-8.
```

Reply:
filed: https://github.com/w3c/baggage/issues/55